pages/council.vue

```diff
@@ -12,8 +12,35 @@
       </div>
 
     </div>
-    <div>
-      <style>.embed-container {position: relative; padding-bottom: 71%; height: 0; max-width: 100%;} .embed-container iframe, .embed-container object, .embed-container iframe{position: absolute; top: 0; left: 0; width: 100%; height: 100%;} small{position: absolute; z-index: 40; bottom: 0; margin-bottom: -15px;}</style><div class="embed-container"><iframe width="900" height="640" frameborder="0" scrolling="no" marginheight="0" marginwidth="0" title="Choropleth Map of Trial Location" src="//edinuniv.maps.arcgis.com/apps/Embed/index.html?webmap=18afc056bff34ae3a7d76fade8cd195d&extent=-5.6973,55.3553,-1.6406,56.674&zoom=true&previewImage=false&scale=true&search=true&searchextent=true&legend=true&disable_scroll=false&theme=dark"></iframe></div>
+    <div style="height: 90%;">
```

Comment:
Has anything changed here or is this just a code format change? Not spotting the edits. Sorry if they are there!

Reply:
No edit here sorry!

Reply:
Cool, thanks for confirming Ruby. Marking as resolved